chain/src/Pos/Chain/Update/Poll/Class.hs

```diff
@@ -348,7 +348,8 @@ instance (MonadPollRead m) =>
         adoptedBVD <- getAdoptedBVData
         case bvs of
             Nothing ->
-                logWarning $ "setAdoptedBV: unknown version " <> pretty bv -- can't happen actually
+                --logWarning $ "setAdoptedBV: unknown version " <> pretty bv -- can't happen actually
+                return ()
```

Comment:
"can't happen actually" would love to have a justification :|

Why have it commented out?